data/sql/updates/pending_db_world/rev_1609118876085874200.sql

```diff
@@ -0,0 +1,10 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1609118876085874200');
+
+DELETE FROM `gameobject` WHERE (`id` = 192011 and `guid` IN (99732, 2134518));
```

Comment:
AND

Reply:
isn't there an "AND" already?

Reply:
He means that your 'and' should be capitalized

Reply:
Oh, thanks @LukasVolgger